src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+const RENTAL_COST = 40;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM = 3;
+const SHORT_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= LONG_TERM) {
+    return RENTAL_COST * days - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return RENTAL_COST * days - SHORT_TERM_DISCOUNT;
+  }
+
+  const allRentalCost = RENTAL_COST * days;
```

Comment:
move your variable above and use it in all cases
```suggestion
  const allRentalCost = RENTAL_COST * days;

  if (days >= LONG_TERM) {
    return allRentalCost - LONG_TERM_DISCOUNT;
  }

  if (days >= SHORT_TERM) {
    return allRentalCost - SHORT_TERM_DISCOUNT;
  }
```